header-verifier/tests/header_verifier_blackbox_tests.rs

```diff
@@ -540,34 +545,45 @@ fn test_change_validator_set() {
         .complete_header_verifier_setup_phase(None);
 
     let operation_hash = ManagedBuffer::from("operation_1");
-    let hash_of_hashes = state.get_operation_hash(&operation_hash);
-
-    for id in 2..4 {
```

Comment:
Please rollback this because here we register 2 validators but in the same epoch with the same bitamp 0x01.